docs/DeveloperGuide.adoc

```diff
@@ -331,32 +331,146 @@ _{More to be added}_
 (For all use cases below, the *System* is the `AddressBook` and the *Actor* is the `user`, unless specified otherwise)
 
 [discrete]
-=== Use case: Delete person
+=== Use case: UC01 - Add an order
 
 *MSS*
 
-1.  User requests to list persons
-2.  AddressBook shows a list of persons
-3.  User requests to delete a specific person in the list
-4.  AddressBook deletes the person
+1.  User key in the order details.
+2.  Delino add the order details.
+3.  Delino displays order added.
 +
 Use case ends.
 
 *Extensions*
 
 [none]
-* 2a. The list is empty.
+* 1a. Delino detects invalid syntax.
+[none]
+** 1a1. Delino shows an error message.
 +
 Use case ends.
 
-* 3a. The given index is invalid.
+[discrete]
+=== Use case: UC02 - Clear all orders
+
+*MSS*
+
+1.  User requests to clear all orders.
+2.  Delino clear all existing orders.
```

Comment:
Maybe Delino can prompt the user to ask if the user really wants to delete all the orders before actually deleting it